src/helpers/env.js

```diff
@@ -166,7 +191,18 @@ export default class EnvManager {
    * @return {*} - env variable value
    */
   get(key) {
-    return this.envMap.get(key);
+    const v = this.envMap.get(key);
+    if (v) return v;
+
+    if (this.runtime === "node") {
+      return process.env[key];
```

Comment:
Redundancy?

Reply:
Not really. `envMap` only has mapped keys. process.env / Deno.env may have more keys.